.eslintrc.js

```diff
@@ -15,6 +20,18 @@ module.exports = {
   },
   overrides: [
     ...overrides,
+    {
+      files: ['.*.js'],
+      rules: {
+        strict: 0,
+      },
+    },
+    {
+      files: ['**/*.md'],
+      rules: {
+        strict: 0,
+      },
+    },
```

Comment:
Separate those two since those should be eventually merged separately to `@netlify/eslint-config-node`.